src/hooks/viewport/useMediaQuery.tsx

```diff
@@ -1,10 +1,11 @@
 import { useCallback, useEffect, useState } from "react";
 
-type MediaType = "isMobile" | "isTablet" | "isLaptop";
+type MediaType = "isMobile" | "isTablet" | "isLaptop" | "isLaptopMin";
 
 const mediaBreakpoints: Record<MediaType, string> = {
   isMobile: "(max-width: 320px)",
   isTablet: "(max-width: 768px)",
+  isLaptopMin: "(max-width: 960px)",
   isLaptop: "(max-width: 1024px)",
```

Comment:
Maybe a longer rename for both would be good.
Something like `isSmallLaptop`, `isLargeLaptop` or `isLaptopSmall`, `isLaptopMedium`.

I'd stay away from two or three letter shortcuts here to make it more clear.

Reply:
[1ec7d1e](https://github.com/sparkletown/sparkle/pull/2973/commits/1ec7d1e6854fb0c34f6633c95e8d4ec95fe9603c)